validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -243,6 +245,17 @@ public static ValidatorClientService create(
                   .propagateTo(validatorClientService.initializationComplete);
               return SafeFuture.COMPLETE;
             })
+        .exceptionally(
+            er -> {
+              Optional<Throwable> maybeCause =
```

Comment:
final, and lets call 'er' something else? `error` and `err` seem to be common, i think `error` is better...